[2024-11-17T02:40:51.917+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-17T02:40:51.935+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: criar_tabelas.inserir_dados_historico manual__2024-11-17T02:40:28.959006+00:00 [queued]>
[2024-11-17T02:40:51.942+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: criar_tabelas.inserir_dados_historico manual__2024-11-17T02:40:28.959006+00:00 [queued]>
[2024-11-17T02:40:51.942+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-11-17T02:40:51.953+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): inserir_dados_historico> on 2024-11-17 02:40:28.959006+00:00
[2024-11-17T02:40:51.963+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1185) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-17T02:40:51.964+0000] {standard_task_runner.py:72} INFO - Started process 1187 to run task
[2024-11-17T02:40:51.966+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'criar_tabelas', 'inserir_dados_historico', 'manual__2024-11-17T02:40:28.959006+00:00', '--job-id', '145', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpi8hxt3rr']
[2024-11-17T02:40:51.967+0000] {standard_task_runner.py:105} INFO - Job 145: Subtask inserir_dados_historico
[2024-11-17T02:40:52.012+0000] {task_command.py:467} INFO - Running <TaskInstance: criar_tabelas.inserir_dados_historico manual__2024-11-17T02:40:28.959006+00:00 [running]> on host bfeda1f9cf11
[2024-11-17T02:40:52.282+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='criar_tabelas' AIRFLOW_CTX_TASK_ID='inserir_dados_historico' AIRFLOW_CTX_EXECUTION_DATE='2024-11-17T02:40:28.959006+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-17T02:40:28.959006+00:00'
[2024-11-17T02:40:52.283+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-11-17T02:40:52.283+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-11-17T02:40:52.284+0000] {logging_mixin.py:190} INFO - Current task name:inserir_dados_historico state:running start_date:2024-11-17 02:40:51.935502+00:00
[2024-11-17T02:40:52.284+0000] {logging_mixin.py:190} INFO - Dag name:criar_tabelas and current dag run status:running
[2024-11-17T02:40:52.284+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-17T02:40:52.512+0000] {logging_mixin.py:190} INFO - Request failed with status code 400
[2024-11-17T02:40:52.513+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-17T02:40:52.524+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-17T02:40:52.525+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=criar_tabelas, task_id=inserir_dados_historico, run_id=manual__2024-11-17T02:40:28.959006+00:00, execution_date=20241117T024028, start_date=20241117T024051, end_date=20241117T024052
[2024-11-17T02:40:52.543+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-11-17T02:40:52.543+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-11-17T02:40:52.544+0000] {logging_mixin.py:190} INFO - Dag name:criar_tabelas queued_at:2024-11-17 02:40:28.974093+00:00
[2024-11-17T02:40:52.544+0000] {logging_mixin.py:190} INFO - Task hostname:bfeda1f9cf11 operator:PythonOperator
[2024-11-17T02:40:52.580+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-17T02:40:52.608+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-17T02:40:52.609+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
